content/intro-to-storybook/react/en/get-started.md

```diff
@@ -67,7 +67,8 @@ If you want to modify the styling, the source LESS files are provided in the Git
 To match the intended design, you'll need to download both the font and icon directories and place its contents inside your `public` folder.
 
 <div class="aside">
-<p>We’ve used <code>svn</code> (Subversion) to easily download a folder of files from GitHub. If you don’t have subversion installed or want to just do it manually, you can grab the folders directly <a href="https://github.com/chromaui/learnstorybook-code/tree/master/public">here</a>.</p></div>
+<p>We’ve used <code>svn</code> (Subversion) to easily download a folder of files from GitHub. If you don’t have subversion installed or want to just do it manually, you can grab the folders directly <a href="https://github.com/chromaui/learnstorybook-code/tree/master/public">here</a>.</p>
+<p>To configure Storybook to serve static content refer to <a href="https://storybook.js.org/docs/configurations/serving-static-files">serving static content</a>.</p></div>
```

Comment:
@orelmax thank you for this small but rather on topic contribution. But i think we can do better. Probably go with something like:

```html
<p>If you want to learn more about serving static content in your Storybook, you can read more about it in <a href="https://storybook.js.org/docs/configurations/serving-static-files">here</a>.</p>
```

Reply:
Yeah, this sounds better. I've finished the whole tutorial but didn't found any mentions of serving statics configuration and thought it'd improve people experience.

Reply:
@orelmax that's beyond of the scope of the tutorial, as it's focus is on informing you in this case how to set it up with a practical example and throughout it mention the core functionalities at your disposal. If you don't mind i'm ok with merging this and i'll do it tomorrow (sunday) if you don't mind.